### DISCUSSION TITLE
Unable to publish apps in Microsoft Store

### DISCUSSION THREAD

**Discussion by WenHong-Lau** [microsoft/WindowsAppSDK#4631] [Dev team discussions]
Hi I am trying to publish my application to the Microsoft Store, it did not pass the validation. I am not able to downgrade as I am using Uno platform's Uno.Sdk and it is using the newer and unsupported version of Microsoft.Windows.SDK.BuildTools (10.0.26100.1).

This is the error
Package acceptance validation error: The package uses an unsupported version of file makepri.exe (10.0.26100.1). The following versions are allowed: < 10.0.10500.0; >= 10.0.10586.0 & < 10.0.11000.0; >= 10.0.14383.0 & < 10.0.14800.0; >= 10.0.15053.0 & < 10.0.15100.0; >= 10.0.16299.0 & < 10.0.16350.0; >= 10.0.17134.0 & < 10.0.17500.0; >= 10.0.17763.0 & < 10.0.18199.0; >= 10.0.18362.0 & < 10.0.18799.0; >= 10.0.19041.0 & < 10.0.19399.0; >= 10.0.20348.0 & < 10.0.21242.0; >= 10.0.22000.194 & < 10.0.22350.0; >= 10.0.22621.0 & < 10.0.25054.0. Please update your Visual Studio build tools and try again.

**Comment by frogcrush** [comment]
Crazy that this is still an issue with 10.0.26100.1742.

**Comment by RDMacLachlan** [comment]
Created Issue #4862 from this discussion. Please go here for investigation and follow-up.
